### ISSUE TITLE
Unknown Crash During Game as a Player

### ISSUE THREAD

**Issue by Yuri-Fanatic** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#550]
**Player Text Document**
----------------------------------------------------------------------------------------------------------
Mono path[0] = 'C:/Program Files (x86)/Steam/steamapps/common/TaleSpire/TaleSpire_Data/Managed'
Mono config path = 'C:/Program Files (x86)/Steam/steamapps/common/TaleSpire/MonoBleedingEdge/etc'
Initialize engine version: 2019.3.0f3 (6c9e2bfd6f81)
[Subsystems] Discovering subsystems at path C:/Program Files (x86)/Steam/steamapps/common/TaleSpire/TaleSpire_Data/UnitySubsystems
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA GeForce RTX 2070 SUPER (ID=0x1e84)
    Vendor:   
    VRAM:     8011 MB
    Driver:   26.21.14.4587
Begin MonoManager ReloadAssembly
- Completed reload, in  0.136 seconds
D3D11 device created for Microsoft Media Foundation video decoding.
<RI> Initializing input.

New input system (experimental) initialized
<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.785300 ms
[SRDebugger] No EventSystem found in scene - creating a default one. 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

DontDestroyOnLoad only works for root GameObjects or components on root GameObjects. 
(Filename:  Line: 582)

-- Noice. Steamworks started fine. SteamID=76561198109650521 PersonalName=Yuri Fanatic BuildId=0 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

DontDestroyOnLoad only works for root GameObjects or components on root GameObjects. 
(Filename:  Line: 582)

WARNING: Shader Unsupported: 'Hidden/PostProcessing/FinalPass' - Pass '' has no vertex shader
WARNING: Shader Unsupported: 'Hidden/PostProcessing/FinalPass' - Pass '' has no vertex shader
WARNING: Shader Unsupported: 'Hidden/PostProcessing/Uber' - Pass '' has no vertex shader
WARNING: Shader Unsupported: 'Hidden/PostProcessing/Uber' - Pass '' has no vertex shader
Waiting for AvailableRegions. State: ConnectedToNameServer Server: NameServer PhotonNetwork.networkingPeer.AvailableRegions False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendManager): Switching from SignedOut to RedeemingSessionIdForConnection 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Disconnected received BackendSocketClient+SwitchServer message whilst in the Disconnected state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Switching from Disconnected to Connecting 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

[WsLog 5/21/2020 8:15:48 PM] The current logging level has been changed to Trace.
---
WsTrace:
set_Level at offset 0 in file:line:column <filename unknown>:0:0
--- 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

[WsLog 5/21/2020 8:15:48 PM] A request to the server:
GET /api/redeem_session HTTP/1.1
User-Agent: websocket-sharp/1.0
Host: prod1.dicewillsuffice.com
Upgrade: websocket
Connection: Upgrade
Sec-WebSocket-Key: 70BYu/gejvxajCY7eHJ9NQ==
Sec-WebSocket-Version: 13
Cookie: session=878922a8-e1d6-4a7c-80ed-b33afac29513

---
WsTrace:
sendHttpRequest at offset 0 in file:line:column <filename unknown>:0:0
--- 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

[WsLog 5/21/2020 8:15:48 PM] A response to this request:
HTTP/1.1 101 Switching Protocols
connection: Upgrade
date: Fri, 22 May 2020 03:16:10 GMT
sec-websocket-accept: DeQ3J7uMBVU/nOhxtihfCSquUOk=
server: Cowboy
upgrade: websocket

---
WsTrace:
sendHttpRequest at offset 0 in file:line:column <filename unknown>:0:0
--- 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Connecting received BackendSocketClient+HasConnected message whilst in the Connecting state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Switching from Connecting to Connected 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

## CONNECTED TO: https://prod1.dicewillsuffice.com 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendManager): RedeemingSessionIdForConnection received BackendManager+WsOpen message whilst in the RedeemingSessionIdForConnection state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendManager): Switching from RedeemingSessionIdForConnection to ConnectionNormal 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Sending 0 broadcast messages that were enqueued before the connection was established 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Sending 2 requests that were enqueued before the connection was established 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BackendSocketConnection: Didnt find the request. Maybe it timed out already: {} 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BackendSocketConnection: Didnt find the request. Maybe it timed out already: {} 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

--------------------------------------------------------------------------------------------------
**Player-prev Text Document**
--------------------------------------------------------------------------------------------------
Mono path[0] = 'C:/Program Files (x86)/Steam/steamapps/common/TaleSpire/TaleSpire_Data/Managed'
Mono config path = 'C:/Program Files (x86)/Steam/steamapps/common/TaleSpire/MonoBleedingEdge/etc'
Initialize engine version: 2019.3.0f3 (6c9e2bfd6f81)
[Subsystems] Discovering subsystems at path C:/Program Files (x86)/Steam/steamapps/common/TaleSpire/TaleSpire_Data/UnitySubsystems
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA GeForce RTX 2070 SUPER (ID=0x1e84)
    Vendor:   
    VRAM:     8011 MB
    Driver:   26.21.14.4587
Begin MonoManager ReloadAssembly
- Completed reload, in  0.091 seconds
D3D11 device created for Microsoft Media Foundation video decoding.
<RI> Initializing input.

New input system (experimental) initialized
<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.329700 ms
[SRDebugger] No EventSystem found in scene - creating a default one. 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

DontDestroyOnLoad only works for root GameObjects or components on root GameObjects. 
(Filename:  Line: 582)

-- Noice. Steamworks started fine. SteamID=76561198109650521 PersonalName=Yuri Fanatic BuildId=0 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

DontDestroyOnLoad only works for root GameObjects or components on root GameObjects. 
(Filename:  Line: 582)

WARNING: Shader Unsupported: 'Hidden/PostProcessing/FinalPass' - Pass '' has no vertex shader
WARNING: Shader Unsupported: 'Hidden/PostProcessing/FinalPass' - Pass '' has no vertex shader
WARNING: Shader Unsupported: 'Hidden/PostProcessing/Uber' - Pass '' has no vertex shader
WARNING: Shader Unsupported: 'Hidden/PostProcessing/Uber' - Pass '' has no vertex shader
Waiting for AvailableRegions. State: ConnectedToNameServer Server: NameServer PhotonNetwork.networkingPeer.AvailableRegions False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendManager): Switching from SignedOut to RedeemingSessionIdForConnection 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Disconnected received BackendSocketClient+SwitchServer message whilst in the Disconnected state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Switching from Disconnected to Connecting 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

[WsLog 5/21/2020 8:13:28 PM] The current logging level has been changed to Trace.
---
WsTrace:
set_Level at offset 0 in file:line:column <filename unknown>:0:0
--- 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

[WsLog 5/21/2020 8:13:28 PM] A request to the server:
GET /api/redeem_session HTTP/1.1
User-Agent: websocket-sharp/1.0
Host: prod1.dicewillsuffice.com
Upgrade: websocket
Connection: Upgrade
Sec-WebSocket-Key: dh4CZ/brCCebcACW0/2Qwg==
Sec-WebSocket-Version: 13
Cookie: session=14f92bf3-4136-4394-9a76-969825f83050

---
WsTrace:
sendHttpRequest at offset 0 in file:line:column <filename unknown>:0:0
--- 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

[WsLog 5/21/2020 8:13:28 PM] A response to this request:
HTTP/1.1 101 Switching Protocols
connection: Upgrade
date: Fri, 22 May 2020 03:13:50 GMT
sec-websocket-accept: wAyD2jqtkFKCxhv2zmQSzn33WRg=
server: Cowboy
upgrade: websocket

---
WsTrace:
sendHttpRequest at offset 0 in file:line:column <filename unknown>:0:0
--- 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Connecting received BackendSocketClient+HasConnected message whilst in the Connecting state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendSocketClient): Switching from Connecting to Connected 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

## CONNECTED TO: https://prod1.dicewillsuffice.com 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendManager): RedeemingSessionIdForConnection received BackendManager+WsOpen message whilst in the RedeemingSessionIdForConnection state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BackendManager): Switching from RedeemingSessionIdForConnection to ConnectionNormal 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Sending 0 broadcast messages that were enqueued before the connection was established 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Sending 2 requests that were enqueued before the connection was established 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BackendSocketConnection: Didnt find the request. Maybe it timed out already: {} 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BackendSocketConnection: Didnt find the request. Maybe it timed out already: {} 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(CampaignSessionManager): OutsideCampaign received CampaignSessionManager+SwitchCampaign message whilst in the OutsideCampaign state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(CampaignSessionManager): Switching from OutsideCampaign to InCampaignOutOfBuildScene 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(CampaignSessionManager): InCampaignOutOfBuildScene received CampaignSessionManager+JoinedBuilderScene message whilst in the InCampaignOutOfBuildScene state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(CampaignSessionManager): Switching from InCampaignOutOfBuildScene to InCampaignInBuildScene 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(CampaignSessionManager): InCampaignInBuildScene received CampaignSessionManager+BoardSummaries message whilst in the InCampaignInBuildScene state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(CampaignSessionManager): InCampaignInBuildScene received CampaignSessionManager+BoardSummaries message whilst in the InCampaignInBuildScene state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Inactive received BoardSessionManager+SwitchToBoard message whilst in the Inactive state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Switching from Inactive to FetchingExistingBoard 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): InitialDisconnected received PhotonConnectionManager+SwitchToRoom message whilst in the InitialDisconnected state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

ConnectToRegion: cae 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnConnectedToMaster 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): InitialDisconnected received PhotonConnectionManager+PhotonConnected message whilst in the InitialDisconnected state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): Switching from InitialDisconnected to Connected 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): Switching from Connected to JoiningRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonCustomRoomPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnCreatedRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): JoiningRoom received PhotonConnectionManager+RoomJoined message whilst in the JoiningRoom state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): Switching from JoiningRoom to InRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): FetchingExistingBoard received BoardSessionManager+JoinedBoard message whilst in the FetchingExistingBoard state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Waiting on buffered rpcs 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnJoinedRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): InRoom received PhotonConnectionManager+RoomJoined message whilst in the InRoom state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Reached deadline. Scheduling fetch 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Fetching board from the backend 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Timer is accurate within 100 nanoseconds 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Trying to claim DataModel.Client(0) 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Switching from FetchingExistingBoard to SettingUp 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of datamodel id by1
IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager.SetupClientAsDataModelClient
clientGuid=627545fc-2c9a-40d6-8152-fd90d9a27a4d clientId=0 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Setting up local DataModel.Client 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of completed pull from1
IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Switching from SettingUp to Active 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(CampaignSessionManager): InCampaignInBuildScene received CampaignSessionManager+BoardSummaries message whilst in the InCampaignInBuildScene state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Active received BoardSessionManager+SwitchToBoard message whilst in the Active state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Switching from Active to TransitioningToLoadADifferentBoard 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Switching from TransitioningToLoadADifferentBoard to FetchingExistingBoard 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): InRoom received PhotonConnectionManager+SwitchToRoom message whilst in the InRoom state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnLeftRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): InRoom received PhotonConnectionManager+LeftRoom message whilst in the InRoom state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): Switching from InRoom to WaitingOnPhotonPostLeavingRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnConnectedToMaster 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): WaitingOnPhotonPostLeavingRoom received PhotonConnectionManager+PhotonConnected message whilst in the WaitingOnPhotonPostLeavingRoom state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): Switching from WaitingOnPhotonPostLeavingRoom to JoiningRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonCustomRoomPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnJoinedRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): JoiningRoom received PhotonConnectionManager+RoomJoined message whilst in the JoiningRoom state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(PhotonConnectionManager): Switching from JoiningRoom to InRoom 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): FetchingExistingBoard received BoardSessionManager+JoinedBoard message whilst in the FetchingExistingBoard state 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Waiting on buffered rpcs 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of datamodel id by1
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager.SetupClientAsDataModelClient
clientGuid=70bb2d12-9372-4fb5-854f-fce1facbab57 clientId=0 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of completed pull from1
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of first change by1
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of completed pull from3
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of datamodel id by3
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager.SetupClientAsDataModelClient
clientGuid=52b2514e-5c4a-42c1-acec-7774e2e264bf clientId=2 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of completed pull from4
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of datamodel id by4
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager.SetupClientAsDataModelClient
clientGuid=1bf50b62-e88f-4adb-ba44-9a1f96b66cb8 clientId=1 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of completed pull from6
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of datamodel id by6
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager.SetupClientAsDataModelClient
clientGuid=b4161e0e-c973-4a7a-9a14-3ee3ca9d0839 clientId=4 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of first change by6
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of first change by3
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of first change by4
IsMe=False 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Reached deadline. Scheduling fetch 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Fetching board from a client (70bb2d12-9372-4fb5-854f-fce1facbab57) 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Timer is accurate within 100 nanoseconds 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Switching from FetchingExistingBoard to SettingUp 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager.RequestDataModelId() 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

=== Torch is meant to be on 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

=== Torch is meant to be on 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

=== Torch is meant to be on 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of completed pull from10
IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: RPC_RespondToDataModelIdRequest from 1 IsMe=False
reCode=0 clientId=3 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager: recieved buffered rpc of datamodel id by10
IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

BoardSessionManager.SetupClientAsDataModelClient
clientGuid=627545fc-2c9a-40d6-8152-fd90d9a27a4d clientId=3 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Setting up local DataModel.Client 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

STATEM(BoardSessionManager): Switching from SettingUp to Active 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

PhotonCallback: OnPhotonPlayerPropertiesChanged 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

OnPhotonPlayerPropertiesChanged Addendum: IsMe=True 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Crash!!!
SymInit: Symbol-SearchPath: '.;C:\Program Files (x86)\Steam\steamapps\common\TaleSpire;C:\Program Files (x86)\Steam\steamapps\common\TaleSpire;C:\WINDOWS;C:\WINDOWS\system32;SRV*C:\websymbols*http://msdl.microsoft.com/download/symbols;', symOptions: 534, UserName: 'Minec'
OS-Version: 10.0.0
C:\Program Files (x86)\Steam\steamapps\common\TaleSpire\TaleSpire.exe:TaleSpire.exe (00007FF7D5D50000), size: 667648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 2019.3.0.40491
C:\WINDOWS\SYSTEM32\ntdll.dll:ntdll.dll (00007FFDC0AC0000), size: 2031616 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\KERNEL32.DLL:KERNEL32.DLL (00007FFDBEB70000), size: 729088 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.778
C:\WINDOWS\System32\KERNELBASE.dll:KERNELBASE.dll (00007FFDBE7C0000), size: 2768896 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\Program Files (x86)\Steam\steamapps\common\TaleSpire\UnityPlayer.dll:UnityPlayer.dll (00007FFD28C10000), size: 26345472 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 2019.3.0.40491
C:\WINDOWS\System32\USER32.dll:USER32.dll (00007FFDBEC30000), size: 1654784 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\System32\win32u.dll:win32u.dll (00007FFDBDCA0000), size: 135168 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\System32\GDI32.dll:GDI32.dll (00007FFDC02F0000), size: 155648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\gdi32full.dll:gdi32full.dll (00007FFDBDCD0000), size: 1654784 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\System32\msvcp_win.dll:msvcp_win.dll (00007FFDBDC00000), size: 647168 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\ucrtbase.dll:ucrtbase.dll (00007FFDBDB00000), size: 1024000 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\ole32.dll:ole32.dll (00007FFDC0190000), size: 1404928 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.693
C:\WINDOWS\System32\combase.dll:combase.dll (00007FFDBEE50000), size: 3366912 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\RPCRT4.dll:RPCRT4.dll (00007FFDC0070000), size: 1179648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.628
C:\WINDOWS\System32\bcryptPrimitives.dll:bcryptPrimitives.dll (00007FFDBDE70000), size: 524288 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\VERSION.dll:VERSION.dll (00007FFDB7990000), size: 40960 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\advapi32.dll:advapi32.dll (00007FFDBF340000), size: 667648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.752
C:\WINDOWS\System32\msvcrt.dll:msvcrt.dll (00007FFDBF4C0000), size: 647168 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 7.0.18362.1
C:\WINDOWS\System32\sechost.dll:sechost.dll (00007FFDC09E0000), size: 618496 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.693
C:\WINDOWS\System32\SHLWAPI.dll:SHLWAPI.dll (00007FFDBF3F0000), size: 335872 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\SETUPAPI.dll:SETUPAPI.dll (00007FFDC0320000), size: 4653056 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\cfgmgr32.dll:cfgmgr32.dll (00007FFDBEAD0000), size: 303104 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\System32\bcrypt.dll:bcrypt.dll (00007FFDBEB40000), size: 155648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.267
C:\WINDOWS\System32\SHELL32.dll:SHELL32.dll (00007FFDBF700000), size: 7225344 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\shcore.dll:shcore.dll (00007FFDC0930000), size: 692224 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\System32\windows.storage.dll:windows.storage.dll (00007FFDBE040000), size: 7856128 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\System32\profapi.dll:profapi.dll (00007FFDBD9B0000), size: 143360 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.693
C:\WINDOWS\System32\powrprof.dll:powrprof.dll (00007FFDBDA00000), size: 303104 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\UMPDC.dll:UMPDC.dll (00007FFDBD980000), size: 65536 (result: 0), SymType: '-deferred-', PDB: ''
C:\WINDOWS\System32\kernel.appcore.dll:kernel.appcore.dll (00007FFDBD990000), size: 69632 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\cryptsp.dll:cryptsp.dll (00007FFDBEB20000), size: 94208 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\OLEAUT32.dll:OLEAUT32.dll (00007FFDC0860000), size: 802816 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\System32\IMM32.dll:IMM32.dll (00007FFDC0030000), size: 188416 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\System32\CRYPT32.dll:CRYPT32.dll (00007FFDBDEF0000), size: 1347584 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.592
C:\WINDOWS\SYSTEM32\OPENGL32.dll:OPENGL32.dll (00007FFD6CCB0000), size: 1400832 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\SYSTEM32\WINMM.dll:WINMM.dll (00007FFDB9410000), size: 147456 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\MSASN1.dll:MSASN1.dll (00007FFDBD9E0000), size: 73728 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\WS2_32.dll:WS2_32.dll (00007FFDBF690000), size: 454656 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\SYSTEM32\WINHTTP.dll:WINHTTP.dll (00007FFDB3570000), size: 983040 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.778
C:\WINDOWS\SYSTEM32\HID.DLL:HID.DLL (00007FFDBC590000), size: 57344 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\GLU32.dll:GLU32.dll (00007FFDB7C70000), size: 180224 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\SYSTEM32\winmmbase.dll:winmmbase.dll (00007FFDB9320000), size: 184320 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\dwmapi.dll:dwmapi.dll (00007FFDBC120000), size: 184320 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.267
C:\WINDOWS\SYSTEM32\dxcore.dll:dxcore.dll (00007FFDBC660000), size: 131072 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\Program Files (x86)\Steam\gameoverlayrenderer64.dll:gameoverlayrenderer64.dll (00007FFD69EA0000), size: 1957888 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 5.86.32.43
C:\WINDOWS\System32\PSAPI.DLL:PSAPI.DLL (00007FFDC0060000), size: 32768 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\system32\uxtheme.dll:uxtheme.dll (00007FFDBBCF0000), size: 626688 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.449
C:\Program Files (x86)\Steam\steamapps\common\TaleSpire\MonoBleedingEdge\EmbedRuntime\mono-2.0-bdwgc.dll:mono-2.0-bdwgc.dll (00007FFD2AB60000), size: 7778304 (result: 0), SymType: '-deferred-', PDB: ''
C:\WINDOWS\system32\mswsock.dll:mswsock.dll (00007FFDBD1F0000), size: 421888 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\MSCTF.dll:MSCTF.dll (00007FFDBFEF0000), size: 1269760 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.752
C:\WINDOWS\SYSTEM32\d3d11.dll:d3d11.dll (00007FFDBB1E0000), size: 2469888 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\SYSTEM32\dxgi.dll:dxgi.dll (00007FFDBC680000), size: 962560 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_95bdb3a23d6478de\nvldumdx.dll:nvldumdx.dll (00007FFDB29A0000), size: 978944 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 26.21.14.4587
C:\WINDOWS\SYSTEM32\cryptnet.dll:cryptnet.dll (00007FFDB6F40000), size: 192512 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\cryptbase.dll:cryptbase.dll (00007FFDBD3C0000), size: 49152 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\WINTRUST.DLL:WINTRUST.DLL (0000021BD4CB0000), size: 376832 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\System32\imagehlp.dll:imagehlp.dll (00007FFDBF560000), size: 118784 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\system32\rsaenh.dll:rsaenh.dll (00007FFDBCD00000), size: 208896 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_95bdb3a23d6478de\nvwgf2umx.dll:nvwgf2umx.dll (00007FFD7E0B0000), size: 41762816 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 26.21.14.4587
C:\WINDOWS\system32\nvspcap64.dll:nvspcap64.dll (00007FFD81260000), size: 2846720 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 3.20.3.63
C:\WINDOWS\SYSTEM32\ntmarta.dll:ntmarta.dll (00007FFDBC9E0000), size: 200704 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\clbcatq.dll:clbcatq.dll (00007FFDBF5E0000), size: 663552 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 2001.12.10941.16384
C:\WINDOWS\system32\wbem\wbemprox.dll:wbemprox.dll (00007FFDB3420000), size: 69632 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\wbemcomn.dll:wbemcomn.dll (00007FFDB2D90000), size: 540672 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\system32\wbem\wbemsvc.dll:wbemsvc.dll (00007FFDB0D00000), size: 81920 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\system32\wbem\fastprox.dll:fastprox.dll (00007FFDB1030000), size: 1052672 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\amsi.dll:amsi.dll (00007FFDB5B50000), size: 86016 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\USERENV.dll:USERENV.dll (00007FFDBD8A0000), size: 151552 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\ProgramData\Microsoft\Windows Defender\platform\4.18.2005.4-0\MpOav.dll:MpOav.dll (00007FFDA3B30000), size: 286720 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 4.18.2005.4
C:\WINDOWS\System32\MMDevApi.dll:MMDevApi.dll (00007FFDB8220000), size: 466944 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\System32\DEVOBJ.dll:DEVOBJ.dll (00007FFDBD7A0000), size: 172032 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.387
C:\WINDOWS\SYSTEM32\AUDIOSES.DLL:AUDIOSES.DLL (00007FFDB8580000), size: 1429504 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\SYSTEM32\wintypes.dll:wintypes.dll (00007FFDB9540000), size: 1388544 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\resourcepolicyclient.dll:resourcepolicyclient.dll (00007FFDBC030000), size: 81920 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\xinput1_4.dll:xinput1_4.dll (00007FFD69880000), size: 69632 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.329
C:\WINDOWS\SYSTEM32\inputhost.dll:inputhost.dll (00007FFDB2E20000), size: 1159168 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\PROPSYS.dll:PROPSYS.dll (00007FFDBA220000), size: 983040 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 7.0.18362.815
C:\WINDOWS\SYSTEM32\CoreMessaging.dll:CoreMessaging.dll (00007FFDBB490000), size: 868352 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\CoreUIComponents.dll:CoreUIComponents.dll (00007FFDB6880000), size: 3317760 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.207
C:\WINDOWS\SYSTEM32\xinput1_3.dll:xinput1_3.dll (0000000000400000), size: 122880 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 9.18.944.0
C:\WINDOWS\SYSTEM32\dcomp.dll:dcomp.dll (00007FFDBB570000), size: 1949696 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\TextInputFramework.dll:TextInputFramework.dll (00007FFDB2F40000), size: 647168 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.693
C:\WINDOWS\system32\Oleacc.dll:Oleacc.dll (00007FFDA7920000), size: 413696 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 7.2.18362.1
C:\Program Files (x86)\Steam\steamapps\common\TaleSpire\TaleSpire_Data\Plugins\steam_api64.dll:steam_api64.dll (0000000052480000), size: 266240 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 4.4.91.85
C:\Program Files (x86)\Steam\steamclient64.dll:steamclient64.dll (00007FFD279B0000), size: 19251200 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 5.86.32.43
C:\WINDOWS\SYSTEM32\IPHLPAPI.DLL:IPHLPAPI.DLL (00007FFDBCF00000), size: 237568 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\Program Files (x86)\Steam\tier0_s64.dll:tier0_s64.dll (00007FFD33800000), size: 1523712 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 5.86.32.43
C:\Program Files (x86)\Steam\vstdlib_s64.dll:vstdlib_s64.dll (00007FFD9C030000), size: 507904 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 5.86.32.43
C:\WINDOWS\SYSTEM32\Secur32.dll:Secur32.dll (00007FFD9FAE0000), size: 49152 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\SSPICLI.DLL:SSPICLI.DLL (00007FFDBD870000), size: 192512 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\DNSAPI.dll:DNSAPI.dll (00007FFDBCF40000), size: 831488 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\System32\NSI.dll:NSI.dll (00007FFDBFE50000), size: 32768 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.449
C:\WINDOWS\SYSTEM32\dhcpcsvc6.DLL:dhcpcsvc6.DLL (00007FFDB66D0000), size: 90112 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\SYSTEM32\dhcpcsvc.DLL:dhcpcsvc.DLL (00007FFDB7A30000), size: 114688 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.815
C:\WINDOWS\SYSTEM32\WINNSI.DLL:WINNSI.DLL (00007FFDB7A90000), size: 45056 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.449
C:\WINDOWS\system32\napinsp.dll:napinsp.dll (00007FFD8BD50000), size: 90112 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\system32\pnrpnsp.dll:pnrpnsp.dll (00007FFD8BD30000), size: 106496 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\system32\wshbth.dll:wshbth.dll (00007FFDB9520000), size: 86016 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\system32\NLAapi.dll:NLAapi.dll (00007FFDBA5D0000), size: 114688 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.693
C:\WINDOWS\System32\winrnr.dll:winrnr.dll (00007FFD8BD00000), size: 57344 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\Windows\System32\rasadhlp.dll:rasadhlp.dll (00007FFDB2D50000), size: 40960 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\System32\fwpuclnt.dll:fwpuclnt.dll (00007FFDB67E0000), size: 487424 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.207
C:\Windows\System32\Windows.UI.dll:Windows.UI.dll (00007FFDB31E0000), size: 1380352 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\gpapi.dll:gpapi.dll (00007FFDBC5A0000), size: 139264 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1
C:\WINDOWS\SYSTEM32\mscms.dll:mscms.dll (00007FFDB57F0000), size: 712704 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\ColorAdapterClient.dll:ColorAdapterClient.dll (00007FFDB57D0000), size: 94208 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\WINDOWS\SYSTEM32\icm32.dll:icm32.dll (00007FFD98DC0000), size: 274432 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.836
C:\Program Files (x86)\Steam\steamapps\common\TaleSpire\MonoBleedingEdge\EmbedRuntime\MonoPosixHelper.dll:MonoPosixHelper.dll (00007FFD8F3A0000), size: 819200 (result: 0), SymType: '-deferred-', PDB: ''
C:\Program Files (x86)\Steam\steamapps\common\TaleSpire\TaleSpire_Data\Plugins\lib_burst_generated.dll:lib_burst_generated.dll (00007FFDABC10000), size: 217088 (result: 0), SymType: '-deferred-', PDB: ''
C:\WINDOWS\SYSTEM32\dbghelp.dll:dbghelp.dll (00007FFDA0600000), size: 2048000 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.18362.1

========== OUTPUTTING STACK TRACE ==================

0x00007FFDBB304E27 (d3d11) CreateDirect3D11SurfaceFromDXGISurface
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FFD2904F75D)
0x00007FFD2904F75D (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FFD2904FA21)
0x00007FFD2904FA21 (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FFD2905AF65)
0x00007FFD2905AF65 (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FFD29103D3D)
0x00007FFD29103D3D (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FFD2910C6AB)
0x00007FFD2910C6AB (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FFD2910DB8B)
0x00007FFD2910DB8B (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FFD2910DC68)
0x00007FFD2910DC68 (UnityPlayer) (function-name not available)
0x00007FFD294275D8 (UnityPlayer) UnityMain
0x00007FFDBEB87BD4 (KERNEL32) BaseThreadInitThunk
0x00007FFDC0B2CE51 (ntdll) RtlUserThreadStart

========== END OF STACKTRACE ===========

A crash has been intercepted by the crash handler. For call stack and other details, see the latest crash report generated in:
 * C:/Users/Minec/AppData/Local/Temp/BouncyRock Entertainment/TaleSpire/Crashes

**Comment by LuPro** [comment]
For future reference, putting texts like from the log into services like pastebin improves readability :)
